latest_cov_build.py

```diff
@@ -22,12 +22,13 @@
 # Download artifacts
 for name in ['target.tar.bz2', 'target.code-coverage-gcno.zip', 'chrome-map.json']:
     url = queue.buildUrl('getLatestArtifact', taskId, 'public/build/{}'.format(name))
-    urlretrieve(url, name)
+    filename = os.path.join('tools', name)
+    urlretrieve(url, filename)
```

Comment:
Nit: no need to define a new variable since the code is so simple, just do `urlretrieve(url, os.path.join('tools', name))`.